data/src/main/java/com/linkedin/util/CustomTypeUtil.java

```diff
@@ -49,18 +49,33 @@ public static String getJavaCustomTypeClassNameFromSchema(final DataSchema schem
 
   public static Class<?> getJavaCustomTypeClassFromSchema(DataSchema schema)
   {
-    final Class<?> bindingClass;
+    Class<?> bindingClass = null;
     final String javaCoercerClassName = getJavaCustomTypeClassNameFromSchema(schema);
 
     if(javaCoercerClassName != null)
     {
+      ClassLoader cl = null;
       try
       {
-        bindingClass = Class.forName(javaCoercerClassName, false, Thread.currentThread().getContextClassLoader());
+        cl = CustomTypeUtil.class.getClassLoader();
```

Comment:
Just do:
`
return Class.forName(javaCoercerClassName, false, CustomTypeUtil.class.getClassLoader());
`

And then catch ClassNotFoundException below to fallback to thread context classloader.

Reply:
Updated